### PR TITLE
Replaced translations for landing app, downgraded MUI Italia package

### PR DESCRIPTION
**Pull request by NicolaGiornetta**
## Short description
Replaced translations for landing app.

## List of changes proposed in this pull request
- Replaced translations for SL, EN, FR and DE languages in landing app
- @pagopa/mui-italia package has been downgraded from 0.9.0 to 0.8.12 as temporary fix for not launching app

## How to test
You need to reinstall packages in pn-landing before testing.
Then switch languages to see the new translations.

### CODE REVIEW

File: packages/pn-landing-webapp/api/data/en.tsx [hunk 1]
```diff
@@ -48,10 +48,10 @@ const onReadClick = () => {
 };
 
 const navigation: INavigationBarProps = {
-  title: "Piattaforma Notifiche",
+  title: "Notifications Platform",
```
Comment (clombardi) [inline packages/pn-landing-webapp/api/data/en.tsx:51]: This title is translated into English and Slovenian, but not in French neither in German, is that on purpose?

**Review by clombardi — COMMENTED** [review]
I'm not sure whether is right to translate the title "Piattaforma Notifiche" into two languages and not into the other two. I would prefer to clarify this detail before approving the PR.

File: packages/pn-landing-webapp/api/data/en.tsx (hunk 1 above)
Comment (NicolaGiornetta) [reply packages/pn-landing-webapp/api/data/en.tsx:51]: Good question and I can't answer. Bring this to the SU.

Comment (clombardi) [reply packages/pn-landing-webapp/api/data/en.tsx:51]: We discussed this issue in the SU, the decision was to merge this PR as is and expose the details about translations in future meetings with other staff linked to PN.

**Review by clombardi — APPROVED** [review]
Works as defined (though not as expected, see resolved conversation), OK to merge.

**Pull request merged**